src/Plivo/Util/ArrayOperations.php

```diff
@@ -35,12 +28,7 @@ public static function removeNull(array $array)
      */
     public static function checkNull(array $array)
     {
-        foreach ($array as $value) {
-            if (is_null($value)) {
-                return true;
-            }
-        }
-        return false;
+        return !in_array(null, $array, true);
```

Comment:
This is the reverse of what the function is intended to do. !in_array(null, $array, true) returns false whenever a null value is found, whereas the expected value is true.

This should be `in_array(null, $array, true)`.